enterprise/core-edge/src/test/java/org/neo4j/coreedge/scenarios/ClusterShutdownIT.java

```diff
@@ -95,24 +97,19 @@ private void shouldShutdownEvenThoughWaitingForLock0( int clusterSize, int victi
             final AtomicInteger numberOfInstancesReady = new AtomicInteger();
             for ( int i = 0; i < NUMBER_OF_LOCK_ACQUIRERS; i++ )
             {
-                txExecutor.execute( new Runnable()
-                {
-                    @Override
-                    public void run()
+                txExecutor.execute( () -> {
+                    try ( Transaction tx = victimDB.beginTx() )
+                    {
+                        numberOfInstancesReady.incrementAndGet();
+
+                        tx.acquireWriteLock( someNode.get() );
+                        Thread.sleep( LONGER_TIME );
```

Comment:
I mean how do we get out of the Thread.sleep. Maybe I am missing something, I just mean that sleeping for a LONGER_TIME is a long time.